library/proc_macro/src/lib.rs

```diff
@@ -34,6 +34,7 @@
 #![feature(rustc_attrs)]
 #![feature(min_specialization)]
 #![feature(strict_provenance)]
+#![feature(utf8_chunks)]
```

Comment:
It says above:

```
// This library is copied into rust-analyzer to allow loading rustc compiled proc macros.
// Please avoid unstable features where possible to minimize the amount of changes necessary
// to make it compile with rust-analyzer on stable.
```

Can we do escaping without using unstable features?

Reply:
Good catch. I've changed this to use stable code only.